pages/src/main/java/uk/gov/android/ui/pages/LandingPage.kt

```diff
@@ -147,7 +147,7 @@ class LandingPageProvider : PreviewParameterProvider<LandingPageParameters> {
 }
 
 @Preview(
-    backgroundColor = 0xFFFBFDF8,
+    backgroundColor = 0xFFFFFFFF,
```

Comment:
Do we want to use the `Color.toHexString()` function here? It probably won't be a string for this value so some logic in the extension function may need to be pulled out

Reply:
We can't:

> An annotation argument must be a compile-time constant

Reply:
wack. Alrighty then